### DISCUSSION TITLE
Stuck with a problem surrounding svg and favicon.

### DISCUSSION THREAD

**Discussion by Banana-Phoney** [chipzoller/hugo-clarity#256] [Q&A] [answered]
I am looking for some help with what to do with troubles i have with svg icons and favicon.

svg icons gives me an 404 not found. the requested URL was not found on this server. Apache/2.4.25 (Debian) server at "mydomain.com" Dort 443
![image](https://user-images.githubusercontent.com/37187278/153069590-c0eb096c-6195-4355-9e0c-34a2be2e0dd9.png)

At this moment i have tried a bog standard setup with Option 1 from the read me. I have not changed anything except the domain name.
Everything looks great when running locally but the problems arise when i upload it to the server.

```
hugo version
hugo v0.92.1-85E2E862+extended windows/amd64 BuildDate=2022-01-27T11:44:41Z VendorInfo=gohugoio
```

I have been looking into it for a bit, but I'm new to all of this and now I'm empty of things to google.

Is there something i have missed? Any tips is much appreciated!

**Comment by chipzoller** [comment, marked as answer]
@onweru

**Reply by onweru** [reply]
It's related to #30. Might be able to fix this one for good. 


Actually I think we fixed it @Banana-Phoney, please check that you have set your icons directory https://github.com/chipzoller/hugo-clarity/blob/1b5b78299b4c410c61c09099e86b8bd45ff445bd/exampleSite/config/_default/params.toml#L97-L98 ... If using a nginx/Apache server please use something else other than the ./icons dir

**Reply by Banana-Phoney** [reply]
Thank you! That worked like a charm!
I should have seen that but must have been blind when looking there.
Again thank you and i have added the Norwegian translation if you want it

[no.txt](https://github.com/chipzoller/hugo-clarity/files/8049231/no.txt)

**Reply by onweru** [reply]
Thanks @Banana-Phoney for the translations.
